Add plot_rgb vignette

jlpalomino opened the pull request:
@lwasser This pull request adds the plot_rgb vignette and updates the CHANGELOG and the README with this new vignette. Fixes #306

lwasser commented:
thank you @jlpalomino will you please update from master today? it looks like there are some conflicts to resolve. if you resolve them locally they shouldn't push to master directly also!

lwasser commented on examples/plot_rgb.py:
```diff
@@ -0,0 +1,168 @@
+"""
+Plot Combinations of Raster Bands Using EarthPy
```
This is a good start. i think we should add a section on using the extent parameter. please look at the parameters and make sure the key ones are covered in the vignette!

jlpalomino replied:
Added this using the crop vignette as a starting point.

lwasser commented on examples/plot_rgb.py:
```diff
@@ -0,0 +1,168 @@
+)
+plt.show()
+
+###############################################################################
```
please look at the build of this. When i build i see a very large white space at the top and bottom of the figure. if you set the figsize() to ensure the height is SMALLER THAN the width it will fix the white space issue. 12,6 seemed to be decent for the size on my machine locally.

<img width="773" alt="Screen Shot 2019-05-24 at 9 21 18 AM" src="https://user-images.githubusercontent.com/7649194/58338643-5c4d0b00-7e05-11e9-9c39-bfbd7fee992c.png">

jlpalomino replied:
Fixed!

lwasser commented:
@jlpalomino just a note that i just merged two other pr's so you will have to update one last time when you address my comments. nothing else should have to be merged today so you should be free and clear!!

jlpalomino commented:
@lwasser I've added the raster extent to plot the polygon on the composite images and modified the size for the multi-plot figure.  This PR is ready for another review.

lwasser commented:
awesome.. thank you @jlpalomino i made a few tiny edits and once this rebuilds i'll merge!!

lwasser commented:
merging this to address https://github.com/earthlab/earthpy/issues/306  once it's built
